mathics/core/definitions.py

```diff
@@ -169,7 +169,14 @@ def __init__(
                 if name.startswith("Global`"):
                     raise ValueError("autoload defined %s." % name)
 
-            self.builtin.update(self.user)
+            # The idea here is that all the symbols loaded in
+            # autoload become converted in builtin.
+            # For some reason, if we do not do this here,
+            # `Export` and `Import` fails.
+            # TODO: investigate why.
```

Comment:
Is it a fair hypothesis that if Export and Import fail, it may be because the are looking inside the builtin rather than the user definitions?

Reply:
It is...